ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -19,6 +19,12 @@ parameters:
   required: true
 - name: CLUSTER_TYPE
   required: true
+- name: CLUSTER_ENABLE_NESTED_VIRT
+  value: "false"
```

Comment:
from what I see, the "true" set in the config & job isn't overriding this?  I'm testing a fix, will push it once i confirm.